src/main/resources/assets/crimson/blockstates/stripped_crimson_wood.json

```diff
@@ -0,0 +1,16 @@
+{
+	"variants": {
```

Comment:
ditto

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.